activerecord/test/cases/migration/columns_test.rb

```diff
@@ -321,6 +321,29 @@ def test_add_column_without_column_name
       ensure
         connection.drop_table :my_table, if_exists: true
       end
+
+      def test_remove_columns_single_statement
+        connection.create_table "my_table" do |t|
+          t.integer "col_one"
+          t.integer "col_two"
+        end
+
+        expected_query_count = if current_adapter?(:SQLite3Adapter)
+                                 24
```

Comment:
24?

Reply:
I was a bit surprised by this myself. The SQLite3 adapter does a lot of work to drop a column:

https://github.com/rails/rails/blob/main/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb#L250-L257

```sql
PRAGMA foreign_keys
PRAGMA defer_foreign_keys
PRAGMA defer_foreign_keys = ON
PRAGMA foreign_keys = OFF
CREATE TEMPORARY TABLE "amy_table" ("id" integer NOT NULL PRIMARY KEY, "col_one" integer DEFAULT NULL, "col_two" integer DEFAULT NULL)
INSERT INTO "amy_table" ("id","col_one","col_two")
                     SELECT "id","col_one","col_two" FROM "my_table"
DROP TABLE "my_table"
CREATE TABLE "my_table" ("id" integer NOT NULL PRIMARY KEY, "col_two" integer DEFAULT NULL)
INSERT INTO "my_table" ("id","col_two")
                     SELECT "id","col_two" FROM "amy_table"
DROP TABLE "amy_table"
PRAGMA defer_foreign_keys = 0
PRAGMA foreign_keys = 1
PRAGMA foreign_keys
PRAGMA defer_foreign_keys
PRAGMA defer_foreign_keys = ON
PRAGMA foreign_keys = OFF
CREATE TEMPORARY TABLE "amy_table" ("id" integer NOT NULL PRIMARY KEY, "col_two" integer DEFAULT NULL)
INSERT INTO "amy_table" ("id","col_two")
                     SELECT "id","col_two" FROM "my_table"
DROP TABLE "my_table"
CREATE TABLE "my_table" ("id" integer NOT NULL PRIMARY KEY)
INSERT INTO "my_table" ("id")
                     SELECT "id" FROM "amy_table"
DROP TABLE "amy_table"
PRAGMA defer_foreign_keys = 0
PRAGMA foreign_keys = 1.
```

Reply:
Oh right. Sqlite has no ALTER table -_-.

I fear this magic number might prove hard to maintain. Maybe just don't assert at all for it?

Reply:
Sure thing. Done in "Skip query count asset on SQLite3"